blaseball_mike/eventually.py

```diff
@@ -9,10 +9,11 @@
 BASE_URL = 'https://api.sibr.dev/eventually/v2'
 
 
-def search(cache_time=5, limit=100, query={}):
+def search(cache_time=5, limit=100, batch_size=100, query={}):
```

Comment:
can you make `batch_size` the last argument in case someone hasn't been calling `query` as a named param?

Reply:
fixed